pytorch_lightning/trainer/trainer.py

```diff
@@ -90,6 +90,7 @@ def __init__(
             gpus: Optional[Union[List[int], str, int]] = None,
             auto_select_gpus: bool = False,
             num_tpu_cores: Optional[int] = None,
+            tpu_id: Optional[int] = None,
```

Comment:
so I can use only one TPU? not several with indexes like GPU?

Reply:
Not as per my knowledge. xla_distributed only supports 1 or 8 cores. We can't selectively choose the cores. 
Ref: https://pytorch.org/xla/release/1.5/index.html#torch_xla.distributed.xla_multiprocessing.spawn

Reply:
tpu_id is not needed...

Reply:
@williamFalcon I have replaced it with `tpu_cores` as you suggested. Valid values are `1`/`8`/`[<1-8>]`